### PR TITLE
px4io: schedule a reboot if showing the police lights

### PR DESCRIPTION
**Pull request by dagar**
This is a response to the various boards on the test rack getting stuck in the px4io "police lights", but I also think it should be better for most users/developers.

We still need to address the fundamental problem of resources on the px4io.

### CODE REVIEW

File: src/modules/px4iofirmware/px4io.c [hunk 1]
```diff
@@ -377,6 +377,10 @@ user_start(int argc, char *argv[])
 	if (minfo.mxordblk < 600) {
 
 		syslog(LOG_ERR, "ERR: not enough MEM");
+
```
Comment (davids5) [inline src/modules/px4iofirmware/px4io.c]: Units?

Comment (dagar) [reply src/modules/px4iofirmware/px4io.c]: Thanks, fixed.

File: src/modules/px4iofirmware/px4io.c [hunk 2]
```diff
@@ -377,6 +377,10 @@ user_start(int argc, char *argv[])
 	if (minfo.mxordblk < 550) {
 
 		syslog(LOG_ERR, "ERR: not enough MEM");
+
+		/* schedule a reboot to give the FMU a chance to update */
```
Comment (davids5) [inline src/modules/px4iofirmware/px4io.c]:
```suggestion
		/* schedule a reboot in 2 seconds to give the FMU a chance to update */
```

Comment (davids5) [reply src/modules/px4iofirmware/px4io.c]: My only questions is will this beat again the FMU boot up. Will the IO be in police lights while the FMU is trying to force program it.

As an alternative we could enable the comms and keep the police lights but reboot on a reboot command.

**Pull request closed without merging**